tests/test_build_images.py

```diff
@@ -416,6 +416,37 @@ def test_packages(self):
         self.assertTrue(self.check_package_output("python --version", "Python 3.10."))
         self.assertTrue(self.is_package_present("pip"))
 
+@pytest.mark.python311
+class TestBIPython311ForArm(BuildImageBase):
+    __test__ = True
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass("python3.11", "Dockerfile-python310", "pip", tag="arm64")
```

Comment:
Apologies not seeing this earlier but this seems like using py3.10 dockerfile?

Reply:
fixed